source/src/test/cs_TestCentral.cpp

```diff
@@ -0,0 +1,146 @@
+/*
+ * Author: Crownstone Team
+ * Copyright: Crownstone (https://crownstone.rocks)
+ * Date: Apr 14, 2021
+ * License: LGPLv3+, Apache License 2.0, and/or MIT (triple-licensed)
+ */
+
+#include <test/cs_TestCentral.h>
+#include <ble/cs_BleCentral.h>
+#include <ble/cs_UUID.h>
+#include <cfg/cs_UuidConfig.h>
+#include <events/cs_EventDispatcher.h>
+#include <util/cs_Utils.h>
+
+TestCentral::TestCentral() :
+	_fwVersionHandle(BLE_GATT_HANDLE_INVALID),
+	_controlHandle(BLE_GATT_HANDLE_INVALID)
+{
+
+}
+
+void TestCentral::init() {
+	EventDispatcher::getInstance().addListener(this);
```

Comment:
Same comment as for the BleCentral. Just call `testCentral.listen();` in the calling scope instead of `testCentral.init();`.